.github/codeql/codeql-config.yml

```
@@ -1,7 +1,7 @@
 paths:
  - 'packages/ag-grid-community/src'
  - 'packages/ag-grid-enterprise/src'
- - 'packages/ag-grid-angular/src'
+ - 'packages/packages/ag-grid-angular/projects/ag-grid-angular/src'
```

I think there is a path error here

```suggestion
 - 'packages/ag-grid-angular/projects/ag-grid-angular/src'
```

Also, does `packages/ag-grid-angular` need to be added too? There is a `package.json` and `project.json` there too

---

Path corrected. No, we're only scanning source not deps